client/notices/index.js

```diff
@@ -133,17 +157,19 @@ const notices = {
 	},
 
 	/**
-	 * Callback handler to clear notices when a user leaves current page
+	 * Callback middleware to clear notices when a user leaves current page
```

Comment:
🙂 I actually prefer calling the page.js functions "handlers" (another popular name is "controller") and keep the "middleware" name for the Redux ones. Too many unrelated things are called middleware these days.

Reply:
fair enough :)

Reply:
```suggestion
	 * Callback handler to clear notices when a user leaves current page
```